### PR TITLE
Updated the notes of -relationship-level search parameter

### PR DESCRIPTION
**Pull request by SudhirPai**
Description
----
Updated the Notes Related to Related Person search parameter '-relationship-level'

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
image

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by SantanuBanerjee** [comment]
> ## Description
> Updated the Notes Related to Related Person search parameter '-relationship-level'
> 
> ## PR Checklist
> * [x]  Screenshot(s) of changes attached before changes merged.
>   image
> * [ ]  Screenshot(s) of changes attached after changes merged and published.

I know it is conditional, but just wondering , if it is okay to still keep the 'required' column for the -relationship-level as 'no',  
as it must be set when searching by patient or identifier.

File: content/millennium/r4/individuals/related-person.md [hunk 1]
```diff
@@ -73,8 +73,9 @@ _Implementation Notes_
 Notes:
 
 - When provided, the `identifier` query parameter must include both a system and a code. Example: `identifier=urn:oid:2.16.840.1.113883.3.13.6|RANDES011E8442-2E86-4A00-8394-EFC6A3434A8F`
-- `-relationship-level` will be required with patient and identifier searches in the future.
+- `-relationship-level` will be required with patient and identifier searches.
 - `-relationship-level` can only be provided with `patient` or `identifier`
+- `-relationship-level` can only be set to `Patient`
```
Comment (sujanb12) [inline content/millennium/r4/individuals/related-person.md:78]: Nit: Missing period in the end.

**Review by alexbeckner1 — APPROVED** [review]

**Comment by exopsRamba** [comment]
> I know it is conditional, but just wondering , if it is okay to still keep the 'required' column for the -relationship-level as 'no',
> as it must be set when searching by patient or identifier. Opinions? @exopsRamba @sujanb12

We are only requiring it on the patient/identifier workflow. The encounter workflow should still operate without it being provided.

File: content/millennium/r4/individuals/related-person.md (hunk 1 above)
Comment (exopsRamba) [reply content/millennium/r4/individuals/related-person.md:78]: We should either remove the period on 76or add them to the others.

File: content/millennium/r4/individuals/related-person.md [hunk 2]
```diff
@@ -73,8 +73,9 @@ _Implementation Notes_
 Notes:
 
 - When provided, the `identifier` query parameter must include both a system and a code. Example: `identifier=urn:oid:2.16.840.1.113883.3.13.6|RANDES011E8442-2E86-4A00-8394-EFC6A3434A8F`
-- `-relationship-level` will be required with patient and identifier searches in the future.
+- `-relationship-level` will be required with patient and identifier searches.
```
Comment (exopsRamba) [inline content/millennium/r4/individuals/related-person.md:76]: patient and identifier should be in '`' to stay consistent.

File: content/millennium/r4/individuals/related-person.md (hunk 1 above)
Comment (Anurag-Balannagari) [reply content/millennium/r4/individuals/related-person.md:78]: removed period for line 76 here 
https://github.com/cerner/fhir.cerner.com/pull/590/commits/e3709c4ee555f818baaa45357eb71e9ca2c828e8#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR76

File: content/millennium/r4/individuals/related-person.md (hunk 2 above)
Comment (Anurag-Balannagari) [reply content/millennium/r4/individuals/related-person.md]: updated here https://github.com/cerner/fhir.cerner.com/pull/590/commits/e3709c4ee555f818baaa45357eb71e9ca2c828e8#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR76

File: content/millennium/r4/individuals/related-person.md (hunk 1 above)
Comment (sujanb12) [reply content/millennium/r4/individuals/related-person.md:78]: +1

**Review by sujanb12 — APPROVED** [review]

**Review by exopsRamba — APPROVED** [review]

**Review by ebbp123 — APPROVED** [review]

**Comment by BS062721** [comment]
Verified on live
Screen Shot 2021-03-18 at 3 39 57 PM

**Pull request merged**